uiowa/uiowa
# Find Text hotfix for Education

Issue cropped up for Find Text on education.

To reproduce: `blt ds --site=education.uiowa.edu`
`drush @education.local uli /admin/find-text`
Search for something like 'programs'
See no results, and no "no results found" message.





# How to test


Follow the reproduce steps above, only get results.
Check a non-education site, perform a text search, see that it still works.
